dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.webBundleSecurity.javaee-1.0.feature

```diff
@@ -0,0 +1,16 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.webBundleSecurity.javaee-1.0
```

Comment:
This name looks odd to me because there is no such feature as `javaee-1.0`. I suggest calling this `io.openliberty.webBundleSecurity1.0-servlet3.0`.

Reply:
Changed to io.openliberty.webBundleSecurity1.0.javaee. Thanks!